### PR TITLE
CHANGELOG: Fix runtime panic that occurs when KeepAlive is called with a Context implemented by an uncomparable type #18893

### PR DESCRIPTION
**Pull request by ktalg**
https://github.com/etcd-io/etcd/pull/18893

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.4.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 
 ## v3.4.36 (TBC)
 
+### Package `clientv3`
+- Fix [runtime panic that occurs when KeepAlive is called with a Context implemented by an uncomparable type](https://github.com/etcd-io/etcd/pull/18893)
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-3.4.md:10]:
```suggestion
- Fix [runtime panic that occurs when KeepAlive is called with a Context implemented by an uncomparable type](https://github.com/etcd-io/etcd/pull/18936)
```

File: CHANGELOG/CHANGELOG-3.5.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 ## Dependencies
 - Bump [golang-jwt/jwt to 4.5.1 to address GO-2024-3250](https://github.com/etcd-io/etcd/pull/18899).
 
+### Package `clientv3`
+- Fix [runtime panic that occurs when KeepAlive is called with a Context implemented by an uncomparable type](https://github.com/etcd-io/etcd/pull/18893)
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-3.5.md:12]:
```suggestion
- Fix [runtime panic that occurs when KeepAlive is called with a Context implemented by an uncomparable type](https://github.com/etcd-io/etcd/pull/18937)
```

**Comment by ahrtr** [comment]
Also please signoff the commit. Read https://github.com/etcd-io/etcd/pull/18951/checks?check_run_id=33536510903

File: CHANGELOG/CHANGELOG-3.5.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 ## Dependencies
 - Bump [golang-jwt/jwt to 4.5.1 to address GO-2024-3250](https://github.com/etcd-io/etcd/pull/18899).
 
+### Package `clientv3`
```
[the remark this answers is not in the document]
Comment (ahrtr) [reply CHANGELOG/CHANGELOG-3.5.md:8]: You did not resolve this comment yet. Please order the items properly

```
### Package `clientv3`
- Fix [runtime panic that occurs when KeepAlive is called with a Context implemented by an uncomparable type](https://github.com/etcd-io/etcd/pull/18937)

### Dependencies
- Bump [golang-jwt/jwt to 4.5.1 to address GO-2024-3250](https://github.com/etcd-io/etcd/pull/18899).

```

[the remark this answers is not in the document]
Comment (ahrtr) [reply CHANGELOG/CHANGELOG-3.5.md:8]: Oops, you already resolved it. Sorry.

**Review by ahrtr — APPROVED** [review]

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @ktalg

**Pull request merged**